src/pages/Login/index.tsx

```diff
@@ -74,7 +74,11 @@ const Login: React.FC = () => {
         </div>
         <div className="Login__actions">
           <div className="Login__actions-container">
-            {userPubkey && !registeredKey && isRegistering ? <SignatureModal /> : <Web3Button />}
+            {userPubkey && !registeredKey && registerMessage ? (
```

Comment:
It looks like there is an edge case that isn't handled.
Steps to reproduce:

1. Connect
2. Sign the message
3. Change your account
4. You are blocked on the login page

We might need a nested ternary here to display a Sign Message button if the user is connected but doesn't have registeredKey